### ISSUE TITLE
Fix dataset page UI

### ISSUE THREAD

**Issue by BastianGrahm** [tira-io/tira#110]
![image](https://user-images.githubusercontent.com/29681076/130982934-519dae72-4653-4559-970c-ae98cb7c8d83.png)

**Comment by BastianGrahm** [comment]
Moved the actions to the back and made them buttons i.e. style the links as buttons.
![image](https://user-images.githubusercontent.com/29681076/131657192-9bdbfde4-aae3-4b00-abbd-fd8f0015a596.png)
Discourse interaction: 
Visited links have blue link color. Is this fine or do we want to revert this?
For links with `.uk-button-primary` (blue color) this will basically make the text invisible since the shade of the visited link and the button color are very similar. For this I added a small CSS fix in the tira-style.css.
